Restoring Backup with an Existing Seed.md

```diff
@@ -0,0 +1,78 @@
+# Restoring Backup with an Existing Seed
+
+If you have already owned a Skywallet to store your coins and have bought a new one to replace it, you can opt to restore the old Skywallet using the new one.
```

Comment:
This might be done in many other situations as well. Please replace it with something more  generic.